.travis.yml

```diff
@@ -19,10 +15,9 @@ cache:
   directories:
   - $HOME/mycache
 before_cache:
-- mv $HOME/virtualenv/python2.7.9 $HOME/mycache/python2.7.9
-- if [[ $TEST_SUITE != "behave" ]]; then mv $HOME/virtualenv/python3.4.2 $HOME/mycache/python3.4.2; fi
+- mv $HOME/virtualenv/python2.7.12 $HOME/mycache/python2.7.12
```

Comment:
Should we put those versions inside variables as well, the way we do it for ETCD, ZK and CONSUL?